tests/perf-system/analyzer/analyzer.py

```diff
@@ -0,0 +1,284 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache 2.0 License.
+
+import pandas as pd  # type: ignore
+
+# pylint: disable=import-error
+from prettytable import PrettyTable  # type: ignore
+import numpy as np
+import matplotlib.pyplot as plt  # type: ignore
+from typing import List
+import sys
+from loguru import logger as LOG
+
+
+SEC_MS = 1000
+
+# Change default log format
+LOG.remove()
+LOG.add(
+    sys.stdout,
+    format="<green>[{time:HH:mm:ss.SSS}]</green> {message}",
+)
+
+
+class Analyze:
+    def __init__(self):
+        self.latency_list = []
+        self.ms_latency_list = []
+
+    def get_req_type(self, df_responses: pd.DataFrame) -> str:
+        return df_responses.iloc[0]["rawResponse"].split(" ")[0]
+
+    def get_latency_at_i(
+        self, df_sends: pd.DataFrame, df_responses: pd.DataFrame, req_id: int
+    ):
+        # will need to handle the re-submission response from submitter when decided
+        return (
+            df_responses.iloc[req_id]["receiveTime"] - df_sends.iloc[req_id]["sendTime"]
+        )
+
+    def check_success(self, df_responses: pd.DataFrame, req_id: int) -> int:
+        req_resp = df_responses.iloc[req_id]["rawResponse"].split("\n")
+        status_list = req_resp[0].split(" ")
+        # if we get a full statues and says ok increase the successful
+        if len(status_list) > 1 and status_list[1][:3] == "200":

```

Comment:
Best to check for "2\d\d", empty success will be 204 for example, rather than 200.